aries_cloudagent/protocols/present_proof/dif/pres_exch_handler.py

```diff
@@ -752,39 +861,31 @@ async def create_vp(
     Return:
         VerifiablePresentation
     """
+    document_loader = profile.context.inject(DocumentLoader)
```

Comment:
I'd make the document loader an input param to create VP. This way you don't have to pass the profile around everywhere